ambry-named-mysql/src/test/java/com/github/ambry/named/MySqlNamedBlobDbTest.java

```diff
@@ -164,6 +167,50 @@ public void testUpdateBlobStateToReady() throws Exception {
     assertEquals("Blob Id is not matched with the record", id, namedBlobRecord.getBlobId());
   }
 
+  @Test
+  public void testListNamedBlobs() throws Exception {
```

Comment:
please add test cases to integration test as well, since integration test will run the real mysql database query.

Reply:
Added the actual test in the MySqlNamedBlobDbIntegrationTest